client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -142,7 +142,7 @@ class CustomContentTypes extends Component {
 
 	renderPortfolioSettings() {
 		const { translate } = this.props;
-		const fieldLabel = translate( 'Portfolio Projects' );
+		const fieldLabel = translate( 'portfolio projects' );
```

Comment:
:disappointed: 34 existing translations will be lost with this change.

Reply:
We should keep the translations as current.